docs/resources/script.md

```diff
@@ -0,0 +1,33 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "coder_script Resource - terraform-provider-coder"
+subcategory: ""
+description: |-
+  Use this resource to run a script from an agent.
+---
+
+# coder_script (Resource)
+
+Use this resource to run a script from an agent.
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `agent_id` (String) The "id" property of a "coder_agent" resource to associate with.
+- `script` (String) The script to run.
+
+### Optional
+
+- `login_before_ready` (Boolean) This option defines whether or not the user can (by default) login to the workspace before it is ready. Ready means that e.g. the script is done and has exited. When enabled, users may see an incomplete workspace when logging in.
```

Comment:
🤔 Why do we have this? I guess we're replacing `startup_script_behavior`? 

Could we do `behavior=blocking`?

Reply:
It is replacing that. The `behavior=blocking` is kinda weird, because there's also `cron`.

I could do `login_behavior=after_start`, but I'd rather just have it as a bool with the key.